all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<ol><li>Usually it's a good idea to keep all the imports in the very first cell. Then you will firstly check whether the required libraries are already installed, and fix the problems before running some computations.</li><li>Consider changing the names because <code>fields</code>  / <code>all_fields</code>  are not self-explanatory</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>